Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```diff
@@ -65,7 +65,8 @@
           Outputs="$(PnpmLinkStamp)">
     <!-- Store the pnpm link command in a property for easy re-use. -->
     <PropertyGroup>
-      <PnpmLinkCommand>pnpm link "$(NodeJsExtensionsNpmPackageSourcePath)"</PnpmLinkCommand>
+      <PnpmLinkCommand Condition="'$([MSBuild]::IsOSUnixLike())' == 'false'">pnpm link "$(NodeJsExtensionsNpmPackageSourcePath)"</PnpmLinkCommand>
+      <PnpmLinkCommand Condition="'$([MSBuild]::IsOSUnixLike())' == 'true'">node -e "const max = 5; for (let i = 0; i !== max; i++) { try { const proc = require('child_process').spawnSync('pnpm', [ 'link', process.argv[1] ]); proc.output.forEach(output => console.log('' + output)); if (proc.error || proc.status !== 0) throw proc; break; } catch (e) { if (i + 1 === max) process.exit(1); } }" "$(NodeJsExtensionsNpmPackageSourcePath)"</PnpmLinkCommand>
```

Comment:
No better way? Just retry?

Reply:
This problem appears inconsistently and only in CI. I have yet to reproduce it reliably. I suspect a timing issue and probably a bug within PNPM, but since I can't give repro instructions there is no use opening an issue about it. So far every time build failed like this, a single retry reliably solved the issue.